### PR TITLE
migrate prometheus off reconciler.Base

### PR DESCRIPTION
**Pull request by n3wscott**
## Proposed Changes

- migrate prometheus off reconciler.Base

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: prometheus/pkg/reconciler/prometheussource.go [hunk 1]
```diff
@@ -20,8 +20,8 @@ import (
 	"context"
 	"fmt"
```
Comment (mattmoor) [inline prometheus/pkg/reconciler/prometheussource.go]: Format Go code:
```suggestion
	"fmt"

```

**Comment by n3wscott** [comment]
/retest

zipkin did not come up

**Pull request merged**